kroxylicious-api/src/test/java/io/kroxylicious/proxy/authentication/SubjectTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.authentication;
+
+import java.util.Set;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class SubjectTest {
+
+    User user1 = new User("name");
+    User user2 = new User("name2");
+
+    FakeUniquePrincipal unique = new FakeUniquePrincipal("name");
+    FakeUniquePrincipal unique2 = new FakeUniquePrincipal("name2");
+    FakeMultiplePrincipal foo = new FakeMultiplePrincipal("foo");
+    FakeMultiplePrincipal bar = new FakeMultiplePrincipal("bar");
+
+    @Test
+    void userIsRequiredRightNow() { // but not eventually
+        Assertions.assertThatThrownBy(() -> new Subject(unique))
+                .hasMessage("A subject with non-empty principals must have exactly one io.kroxylicious.proxy.authentication.User principal.");
+    }
+
+    @Test
+    void uniquenessIsEnforced() { // but not eventually
```

Comment:
I'm not following why `uniquenessIsEnforced` has the `but not eventually` comment? I think that only applies to `userIsRequiredRightNow ` or am I missing something?

Reply:
Good spot, yes, this is just a result of my copy and paste.